libs/platform/src/lib/components/table/table.component.html

```diff
@@ -72,6 +72,7 @@
                 [id]="id+'__'+column.name"
                 [attr.aria-colindex]="colIdx"
                 [class.fd-table__cell--fixed]="_freezableColumns?.includes(column.name)"
+                [class.fd-table__cell--fixed-last]="column.name === freezeColumnsTo"
```

Comment:
don't we need to consume the version of Fundamental Styles where `fd-table__cell--fixed-last` was introduced? I don't see its styles currently: 
![image](https://user-images.githubusercontent.com/53509521/121298249-2d812f00-c911-11eb-9ddb-6b3ff260362d.png)
also the table looks broken

Reply:
We do, you just cannot see it on netlify deploy due to the cached npm packages used by netlify (where old fundamental-styles library used).

![image](https://user-images.githubusercontent.com/20265336/121310605-328faf80-c90c-11eb-8b63-84eb3a94bca3.png)

Broken layout has own PR #5614. Text truncation will be fixed in scope of table refactoring PR (it's in progress...).